Eplant/views/ChromosomeViewer/Viewer/GeneInfoPopup.tsx

```diff
@@ -0,0 +1,162 @@
+// -------
+// IMPORTS
+// -------
+
+import React, { FC, useEffect, useState } from 'react'
+import Draggable from 'react-draggable'
+
+import GeneticElement from '@eplant/GeneticElement'
+import arabidopsis from '@eplant/Species/arabidopsis'
+import {
+  useActiveGeneId,
+  useGeneticElements,
+  useSetGeneticElements,
+} from '@eplant/state'
+import CloseIcon from '@mui/icons-material/Close'
+import { Typography } from '@mui/material'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import IconButton from '@mui/material/IconButton'
+import Popover from '@mui/material/Popover'
+import useTheme from '@mui/material/styles/useTheme'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableRow from '@mui/material/TableRow'
+
+import { GeneItem } from '../types'
+
+// TYPES
+interface GeneInfoPopupProps {
+  gene: GeneItem
+  open: boolean
+  anchorOrigin: number[]
+}
+
+//----------
+// COMPONENT
+//----------
+const GeneInfoPopup: FC<GeneInfoPopupProps> = (props) => {
+  const [open, setOpen] = useState(props.open)
+  const [gene, setGene] = useState<GeneItem>(props.gene)
+  // Global State
+  const [activeGeneId, setActiveGeneId] = useActiveGeneId()
+  const geneticElements = useGeneticElements()
+  const setGeneticElements = useSetGeneticElements()
+  const theme = useTheme()
+
+  useEffect(() => {
+    if (props.gene != gene) {
+      setOpen(props.open)
+      setGene(props.gene)
+    }
+  }, [props])
+  // --------------
+  // EVENT HANDLERS
+  // --------------
+  const handleClose = () => {
+    setOpen(false)
+  }
+  const handleLoadGeneClick = (event: React.MouseEvent<HTMLElement>) => {
+    if (gene != null) {
+      const geneticElement = new GeneticElement(
+        gene.id,
+        gene.annotation,
+        arabidopsis,
+        gene.aliases
+      )
+      const newGeneticElements = geneticElements[0]
+      newGeneticElements.push(geneticElement)
+      setGeneticElements(newGeneticElements)
+      setActiveGeneId(geneticElement.id)
```

Comment:
```suggestion
      setGeneticElements([...geneticElements, geneticElement])
      setActiveGeneId(geneticElement.id)
```
This will update the state with a new memory reference which will trigger a refresh. With this change you can also remove the `activeGeneId` from the dependency array of the Collections component (I can't think of an instance where the list of genes needs updating on active gene change but I could be wrong on this).